Dockerfile-buildimage

```diff
@@ -0,0 +1,38 @@
+# use this image as your build container to collect the conda dependencies
```

Comment:
This is a docker image to create the `environment-moveaps-<platform>.yml` file. This process should be simplified by the MoveApps-SDK in the future.